### PR TITLE
Change the apostrophe on Kip's description to quotes

### LINKED ISSUE #239: [BUG]: Most recent PR #235 breaks `jekyll serve`
## I'm submitting a ...
  - [x] bug report
  - [ ] feature request
  - [ ] question

## Expected Behavior
Running `jekyll serve` from the base directory of the repo launches the website locally.

## Actual Behavior
The website fails to build due to a missing key.

Here's the full terminal output:
```
Configuration file: /home/ooblack/projects/arfc.github.io/_config.yml
            Source: /home/ooblack/projects/arfc.github.io
       Destination: /home/ooblack/projects/arfc.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
Traceback (most recent call last):
	27: from /home/ooblack/gems/bin/jekyll:23:in `<main>'
	26: from /home/ooblack/gems/bin/jekyll:23:in `load'
	25: from /home/ooblack/gems/gems/jekyll-4.2.1/exe/jekyll:15:in `<top (required)>'
	24: from /home/ooblack/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	23: from /home/ooblack/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	22: from /home/ooblack/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	21: from /home/ooblack/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	20: from /home/ooblack/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	19: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
	18: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	17: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `each'
	16: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	15: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/commands/build.rb:36:in `process'
	14: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/commands/build.rb:65:in `build'
	13: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:28:in `process_site'
	12: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/site.rb:78:in `process'
	11: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/site.rb:181:in `read'
	10: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/reader.rb:19:in `read'
	 9: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/readers/data_reader.rb:22:in `read'
	 8: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/readers/data_reader.rb:40:in `read_data_to'
	 7: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/readers/data_reader.rb:40:in `each'
	 6: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/readers/data_reader.rb:48:in `block in read_data_to'
	 5: from /home/ooblack/gems/gems/jekyll-4.2.1/lib/jekyll/readers/data_reader.rb:70:in `read_data_file'
	 4: from /home/ooblack/gems/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:157:in `load_file'
	 3: from /home/ooblack/gems/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:157:in `open'
	 2: from /home/ooblack/gems/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:157:in `block in load_file'
	 1: from /home/ooblack/gems/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `load'
/home/ooblack/gems/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/home/ooblack/projects/arfc.github.io/_data/alumni.yml): did not find expected key while parsing a block mapping at line 408 column 3 (Psych::SyntaxError)
```
The referenced `.yml` file was changed in #235. I tested the previous version of the code from #233 and did not run into this bug. My suspicion is that there's some issue with the use of commas in that throws off the parser. 

## Steps to Reproduce the Problem

  1. Clone this repo and make sure [you get all the necessary dependencies](https://arfc.github.io/manual/guides/website)
  1. Move the the base directory of the `source` branch of the repo
  1. Run `jekyll serve`

## Specifications

  - Version: Jekyll 4.2.1, Ruby 2.7.2p137
  - Platform: Ubuntu 21.04

## How can this issue be closed?
This issue can be closed when a PR that fixes the bug exists.

### PR DESCRIPTION
**Pull request by nsryan2**
## Summary of changes

This PR is to Closes #239,  the issue Olek noticed where the alumni.yml file had a bug preventing his jekyll from rendering the site locally by changing the apostrophe's on Kip's description to quotations. Madicken, and Olek suspected this in his issue too, pointed out that the possessive in the description was ending the string prematurely. Quotes should be sufficient to work around this issue.

Something I'm still not sure of is how Nataly and I were able to render the site with `jekyll serve` on our machines. @yardasol is there some config you have such that a breaking change prevents the whole site from rendering? I haven't specifically configured jekyll on my machine, so I should be running on the defaults. This might be a good thing to create space for in the guides.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #239 

## Associated Developers

- Dev: @yardasol 

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Review by munkm — APPROVED** [review]

**Comment by munkm** [comment]
This looks good to me! Before merging I'd prefer that @yardasol can confirm that he is able to build it locally.

**Comment by munkm** [comment]
I confirm that I can build locally with this update.

File: _data/alumni.yml [hunk 1]
```diff
@@ -409,7 +409,7 @@
   pic: /img/people/kleimenhagenk.jpg
   dates: 2019-2021
   role: Undergraduate Research Assistant
-  description: 'Kip was an undergraduate student in the NPRE department at UIUC, he's currently pursuing his master’s degree in nuclear engineering.'
+  description: "Kip was an undergraduate student in the NPRE department at UIUC, he's currently pursuing his master’s degree in nuclear engineering."
```
Comment (yardasol) [inline _data/alumni.yml:412]: +1

**Review by yardasol — APPROVED** [review]
Hi @nsryan2,
Thanks for the PR! I can confirm this PR fixes the bug describes in #239. Sorry for the late response on this.
- [x] Read the PR description
- [x] Read through all the changes, considering the following questions.
  - [x] Is there anything you can praise about this PR? Start with
        praise.
  - [x] If there is no continuous integration, check out the branch
        locally, build, and run the tests.
  - [x] Do the tests pass on your machine?
  - [x] Is the PR free of random cruft (built files, `.swp` files,
        etc.)?
  - [x] Do all files in the PR belong in the repository?
  - [x] Make a review, leaving kind comments and suggesting changes
        where needed (to resolve the above).
  - [x] Has the author resolved all of the comments and suggestions
        in your review?
- [x] When you approve of the PR, merge and close it.
- [x] Does this PR close an issue? If so, be sure to descriptively
      close this issue when the PR is merged
- [x] Thank the author for their contribution.

**Pull request merged**